wiki/604838-SubmissionDashboards.md

```diff
@@ -13,6 +13,42 @@ This section provides dashboards to the person who submit NLP Sandbox tools for
 - You can cancel a submission tthat has not started yet.
 - Only submissions that have successfully completed are counted toward your (or your team's) submission quota.
 - If you can't figure out why your submission failed, please [open a ticket].
+- The SubmissionLogs column contains information of the workflow that is ran per submission.  It doens't actually contain logs generated by your submissions.  If your submission is INVALID or REJECTED, you will have to request for logs.
+
+### Submission Status Definitions
+
+We use a the i2b2 dataset to validate the format of the models submitted.
```

Comment:
It's a bit confusing that the i2b2 dataset is used for validation as described by this sentence. The user does not really care about this distinction. Not having a column "i2b2 status" is confusing. Here the i2b2 dataset plays two roles that I would have preferred to decouple: 1) validation and 2) full-fledged dataset. We can discuss this on Monday.